bin/common.ts

```diff
@@ -24,3 +31,38 @@ export const runInQueueStream = async <T>(
   await insertQueue.drained();
   console.log('update finished with a total of: ', insertCount);
 };
+
+type AllowedZodTypes = z.ZodString | z.ZodBoolean;
+
+export const zodToParseArgs = <T extends z.ZodRawShape>(
```

Comment:
Had fun with Zod. Not bulletproof but just enough to work with `parseArgs` specifically.

Defining node arguments can now be based solely on the Zod schema.